### ISSUE TITLE
app is only using my cpu to process the song

### ISSUE THREAD

**Issue by TeamBrainStorm** [ace-step/ACE-Step#165]
the web app launches but only proccess via my cpu it does not use my 3090 gpu. task manager shows my cpu is at 100% and gpu is at 1%  

I use this command to via command prompt 

acestep --checkpoint_path /path/to/checkpoint --port 7865 --device_id 1 --share true 

my system does have integrated graphics as well here is my system info: 
 OS windows 11

cpu: intel ultra 7 
gpu: nvidia 3090 

am I doing something wrong?

Thanks in advance!!!

**Comment by TeamBrainStorm** [comment]
here is my cmd log:

* Running on local URL:  http://127.0.0.1:7865

Could not create share link. Please check your internet connection or our status page: https://status.gradio.app.
2025-05-13 13:02:51.799 | WARNING  | acestep.pipeline_ace_step:__call__:1613 - Checkpoint not loaded, loading checkpoint...
2025-05-13 13:02:51.799 | INFO     | acestep.pipeline_ace_step:load_checkpoint:157 - Download models from Hugging Face: ACE-Step/ACE-Step-v1-3.5B, cache to: /path/to/checkpoint
Fetching 14 files: 100%|███████████████████████████████████████████████████████████████████████| 14/14 [00:00<?, ?it/s]
C:\Users\Owner\.conda\envs\ace_step\lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-05-13 13:02:59.748 | INFO     | acestep.pipeline_ace_step:__call__:1619 - Model loaded in 7.95 seconds.
2025-05-13 13:02:59.849 | INFO     | acestep.pipeline_ace_step:text2music_diffusion_process:1006 - cfg_type: apg, guidance_scale: 15, omega_scale: 10
2025-05-13 13:02:59.861 | INFO     | acestep.pipeline_ace_step:text2music_diffusion_process:1220 - start_idx: 15, end_idx: 45, num_inference_steps: 60
  0%|

**Comment by akande-ogundimu** [comment]
I had the same issue. I concluded it must had happened, because i wrongfully installed the wrong pytorch or the cpu only version. Solution was to:

1.  Uninstall the Current PyTorch CPU Version: 
2. Open your command prompt with your virtual environment (venv) activated, and run: `pip uninstall torch torchvision torchaudio`
3. Determine Your NVIDIA Driver Version and Compatible CUDA Version via `nvidia-smi`
4. Go to the official PyTorch website: https://pytorch.org/get-started/locally/
5. In your activated virtual environment's command prompt, paste and run the command generated by the PyTorch website.
done.

**Comment by TeamBrainStorm** [comment]
Thank you so much for the feedback!!!!

I followed you steps and am stuck on step 4. i do not see a command for my cuda version...

here is my nvidia version 
| NVIDIA-SMI 576.40                 Driver Version: 576.40         CUDA Version: 12.9     |
on the link you sent me there is only up to 12.8 for pytorch. 

any idea what version i should install?

Thanks for all the help :)

**Comment by OoAnikioO** [comment]
i use pinokio, i think i have the same cpu usage, how to change to the gpu with pinokio ?  thanks
